docs/nodes/compute/advanced/local-testing.md

```diff
@@ -0,0 +1,99 @@
+# Running Instances on the Local Environment
+
+Aleph.im supports two popular and high-performance hypervisors for virtualization: Firecracker and Qemu. This guide outlines the steps to run instances on your local machine for development and testing purposes.
+
+## 0. Prerequisites
+
+- **SSH Keys:** Ensure you have a valid SSH key pair set up. You'll need the public key to enable SSH access to the instances.
+
+## 1. Configuration
+
+### **Configure Allocation Token and SSH Keys:**
+
+- Edit the `/etc/aleph/supervisor.env` file and add the following lines:
+
+  ``` bash
+  # Encrypted allocation token hash in sha1 format
+  ALEPH_VM_ALLOCATION_TOKEN_HASH=YOUR_ALLOCATION_TOKEN_HASH
+
+  # List of authorized SSH public keys (replace "YOUR_SSH_PUBLIC_KEY" with your actual key)
+  ALEPH_VM_DEVELOPER_SSH_KEYS=["YOUR_SSH_PUBLIC_KEY"]
+  ```
+
+- **Important:** Replace `YOUR_ALLOCATION_TOKEN_HASH` with your actual encrypted allocation token and `YOUR_SSH_PUBLIC_KEY` with your public SSH key.
+
+
+### **Enable Developer SSH Keys (Optional):**
+
+By default, SSH access to instances is only enabled to the user that creates the VM for security reasons. To enable it for development, you have two options:
+
+#### a) Modifying the Systemd Service File:
```

Comment:
This seems more complex than running the service manually. Why put it first ?

Reply:
I have put this as first because some users installs the aleph-vm package instead clone the repo. So the fast way to modify it is to modify the service instead run the program locally.